hack/update-docs-and-licenses.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Copyright (c) Codesphere Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+IFS=$'\n\t'
+
+here=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+root=$(cd "$here/.." && pwd)
+
+echo "Working directory: $root"
+
+cd "$root"
+
+echo "1/2: Generating docs"
+if command -v make >/dev/null 2>&1; then
```

Comment:
this is duplicated, and honestly I'm not sure how much benefit it gives over just having the make command fail?